tests/functional/NotificationTargetTicket.php

```diff
@@ -360,6 +360,52 @@ public function testTimelineTag()
             ]
         ];
 
+        //add second test for post-only
+        //simulate that is a self-service (because of one profil with interface = helpdesk)
+        //he can't see private because of self-service who has no right TILFollowup::SEEPRIVATE
+        $basic_options = [
+            'additionnaloption' => [
+                'usertype' => NotificationTarget::GLPI_USER,
+                'is_self_service' => true,
+                'show_private'    => false,
+            ]
+        ];
```

Comment:
This is exactly the same test as the previous one.

Reply:
fixed